README.md

```diff
@@ -185,6 +185,14 @@ Note that although the annotation containing router configuration for each of th
 | deis-router          | workerProcesses  | `string`   | `auto` (number of CPU cores) | Number of worker processes to start. |
 | deis-router          | workerConnections | `integer` | `768`         | Maximum number of simultaneous connections that can be opened by a worker process. |
 | deis-router          | defaultTimeout   | `integer`  | 1300          | Default timeout value in seconds.  Should be greater than the front-facing load balancer's timeout value. |
+| deis-router          | useGzip          | `boolean`  | `true`        | Whether to use gzip compression. |
```

Comment:
Not using confd at all currently.  But the reason for the existence of `useGzip` is that I initialize a nested `GzipConfig` with all the defaults gzip settings.  Since I initialize that with some default state, the nested `GzipConfig` is _never_ `nil`.  Hence, a separate switch to turn the whole thing on or off.

It might be possible to allow an operator to `nil` out the whole nested `GzipConfig`.  For my own edification, I am going to try that, but I'm not sure it's a more intuitive means of disabling gzip than by setting `"useGzip": false`.